01/LaneHesser/wordvalue.py

```diff
@@ -0,0 +1,57 @@
+from string import ascii_uppercase
+
+from data import DICTIONARY, LETTER_SCORES
+
+
+def load_words():
+    """Load dictionary into a list and return list"""
+
+    with open(DICTIONARY) as f:
+        return [
+            word.strip()
+            for word in f.read().split()
+        ]
+
+
+def calc_word_value(word: str):
+    """Calculate the value of the word entered into function
+    using imported constant mapping LETTER_SCORES"""
+
+    value = 0
+
+    for letter in word:
+        letter = letter.upper()
+        if letter in ascii_uppercase:
+            value += LETTER_SCORES[letter]
```

Comment:
This works here, if a letter would not be in the `LETTER_SCORES` dict, you could use `LETTER_SCORES.get(letter, 0)`